### DISCUSSION TITLE
Equipo para la Hackaton

### DISCUSSION THREAD

**Discussion by Matiasox** [Launch-X-Latam/MisionFrontEnd#583] [Team Posts]
Hola compañeros explorers voy un poco atrasado en esta ultima etapa backend y quería saber si a alguien le falta equipo. :D

**Comment by Johamaster04** [comment]
Si necesitan a alguien en su grupo pueden agregarme con el correo lx11667@innovaccion.mx

**Comment by Bartt0** [comment]
hola también busco equipo correo lx00284@innovaccion.mx

**Comment by neocitylab** [comment]
Hola , si necesitan a alguien estoy disponible, mi correo es lx10115@innovaccion.mx

**Comment by Ivanhdzsty** [comment]
Hola, tambien estoy en busqueda de equipo mi correo es lx00068@innovaccion.mx, gracias

**Comment by AbnerHub** [comment]
Hola yo por favor 
lx02522@innovaccion.mx

**Comment by jlnunezobando** [comment]
Busco equipo lx08651@innovaccion.mx

**Comment by Andersonm22** [comment]
Encontraron equipo?, si aun necestitan gente yo estoy disponible  lx10407@innovaccion.mx

**Comment by joseandresgavilanes** [comment]
Busco equipo  ***@***.***
